src/ClimateDynamicsComponent.jl

```diff
@@ -7,36 +7,33 @@
     # Average global temperature
     temp = Variable(index=[time])
 
-    # LifeTempConst
-    LifeTempConst = Parameter()
+    # lifetempconst
+    lifetempconst = Parameter(default = -42.7421952811)
```

Comment:
This gets computed by [this](https://github.com/fund-model/fund/blob/next/calibration/climate/fund_cs_calibration.ipynb), so we should move it into a file again.